build-system/tasks/visual-diff.js

```diff
@@ -127,6 +127,22 @@ function setPercyBranch() {
   }
 }
 
+/**
+ * Set the branching point's SHA to an env variable.
+ *
+ * This will let Percy determine which build to use as the baseline for this new
+ * build.
+ *
+ * Does not do anything for --master builds, since master builds are always
+ * built on top of the previous commit (we use the squash and merge method for
+ * pull requests.)
+ */
+function setPercyTargetCommit() {
+  if (!argv.master) {
```

Comment:
I take it this will work for local dev too?

Reply:
Yep, I don't see why it would be any different